.github/workflows/check_examples.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - id: inspect-examples
         name: Inspect Examples
         run: |
+          npm install markdownlint-cli2 --global
```

Comment:
Create package.json with "npm init" in the project. Add by " npm install markdownlint-cli2". Then the instructions will be simply just "npm install".

Don't install anything globally. I'd think you know that from Python already.

Reply:
That will make it reproducible by others.

Reply:
Done